src/ontology/uberon.Makefile

```diff
@@ -636,18 +637,41 @@ $(TMPDIR)/external-disjoints.owl: components/external-disjoints.obo
 	$(ROBOT) convert -i $< -f owl -o $@
 .PRECIOUS: $(TMPDIR)/external-disjoints.owl
 
+# All the checks below need the taxslim-disjoint-over-in-taxon.owl,
+# which should have been downloaded into mirror/ncbitaxondisjoints.owl
+# by the standard ODK import pipeline. So ideally, all the checks below
+# should just have to depend on mirror/ncbitaxondisjoints.owl. But the
+# QC checks are run under MIR=false IMP=false, so while the rule would
+# be invoked by Make it would not result in the mirror being downloaded
+# if it is not already available.
+# So instead, we make the checks depend on another file, and here we
+# 1. force the download of the mirror if it is not already there;
+# 2. create a link to that mirror.
+$(TMPDIR)/taxslim-disjoint-over-in-taxon.owl:
+	if [ ! -f $(MIRRORDIR)/ncbitaxondisjoints.owl ]; then \
+		$(MAKE) $(MIRRORDIR)/ncbitaxondisjoints.owl MIR=true IMP=true ; \
+	fi && \
+	ln -f -s ../$(MIRRORDIR)/ncbitaxondisjoints.owl $@
```

Comment:
cool idea tp save disc space. Hopefully this works on windows systems!

Reply:
Didn’t test, but the Unix emulation layer in Windows (upon which the Windows version of Docker is based, I believe) would do a very poor job if it couldn’t handle something as basic and common as a symbolic link.